lib/adrian/directory_queue.rb

```diff
@@ -36,8 +36,8 @@ def pop_item
 
     def push_item(value)
       item = wrap_item(value)
-      item.move(available_path)
       item.touch
+      item.move(available_path)
```

Comment:
I think we should follow an opt-in approach with this change as discussed in [this thread](https://zendesk.slack.com/archives/G014A8F5STD/p1639678898038200?thread_ts=1639677115.038000&cid=G014A8F5STD) That will help minimize the risk to changing the gem without making it too complicated to arturo.

Reply:
For sure, thanks for the reference.